src/main/java/mekanism/common/tile/transmitter/TileEntityTransmitter.java

```diff
@@ -50,6 +52,10 @@ public void onWorldJoin()
 		{
 			TransmitterNetworkRegistry.registerOrphanTransmitter(getTransmitter());
 		}
+		else if(lastClientNetwork != null)
```

Comment:
What exactly does this solve? The network should be synced to the client via packet at the end of the server tick.